code/modules/defenses/defenses.dm

```diff
@@ -33,6 +33,15 @@
 	var/placed = 0
 	var/obj/item/defenses/handheld/HD
 
+	// Defence Laptop
+	var/nickname = ""
+	var/encryptable = TRUE
+	var/obj/item/device/sentry_computer/linked_laptop = null
```

Comment:
you'll want to null this out on destroy; code doc comments for these would help too